.editorconfig

```diff
@@ -10,3 +10,6 @@ root = True
 indent_size = 4
 indent_style = space
 
+ij_kotlin_allow_trailing_comma_on_call_site=true
+ij_kotlin_allow_trailing_comma=true
```

Comment:
Just to close the loop, did we need to manually enable these 2 new rules?

Reply:
Yes. We need to explicitly allow trailing commas. Without it, the enabled rule by default is to not allow them and it would raise warnings for our existing trailing commas.